docker-compose-dev.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+services:
+
+  web:
+    extends: worker
+    command: ./proc-start web
+    ports:
+      - "3000:3000"
+
+  worker:
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
+    command: ./proc-start worker
+    volumes:
+      - bundle:/home/app/bundle
+      - .:/home/app/src
+    environment:
+      - DATABASE_URL=mysql2://root:secret@mariadb/development?encoding=utf8
+      - REDIS_URL=redis://redis:6379
+      - QUEUE=foodsoft_notifier
+      - TEST_DATABASE_URL=mysql2://root:secret@mariadb/test?encoding=utf8
+
+  mailcatcher:
```

Comment:
i see not configuration sending the mails to this container

Reply:
Took same approach as before, where current configuration detects docker environment variables and [connects to mailcatcher](https://github.com/foodcoops/foodsoft/blob/master/config/environments/development.rb.SAMPLE#L42). Later we can change this to a proper smtp url.